code/modules/religion/sects/shadow_sect.dm

```diff
@@ -60,6 +64,80 @@
 	icon_state = "shadow-obelisk"
 	anchored = FALSE
 	break_message = "<span class='warning'>The Obelisk crumbles before you!</span>"
+	var/heal_delay = 20
+	var/last_heal = 0
+	var/spread_delay = 45
+	var/last_spread = 0
+
+/obj/structure/destructible/religion/shadow_obelisk/Initialize(mapload)
+	. = ..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/destructible/religion/shadow_obelisk/LateInitialize()
+	. = ..()
```

Comment:
theres no need to late initialise this if you aren't doing anything that needs late initialisation. Late initialisation runs after the map has loaded. If this item isn't included in a map, then late initialise runs instantly.

Reply:
all set